freqtrade/freqtradebot.py

```diff
@@ -921,8 +921,7 @@ def handle_timedout_limit_buy(self, trade: Trade, order: Dict) -> bool:
         # cancel_order may not contain the full order dict, so we need to fallback
         # to the order dict aquired before cancelling.
         # we need to fall back to the values from order if corder does not contain these keys.
-        trade.amount = order['amount'] - safe_value_fallback(corder, order,
-                                                             'remaining', 'remaining')
+        trade.amount = safe_value_fallback(corder, order, 'filled', 'filled')
```

Comment:
```suggestion
        trade.amount = filled_value
```
-- see assignment for this var above